plugins/amazonq/codetransform/jetbrains-community/src/software/aws/toolkits/jetbrains/services/codemodernizer/panels/BuildProgressStepDetailsPanel.kt

```diff
@@ -168,7 +168,8 @@ class BuildProgressStepDetailsPanel : JPanel(BorderLayout()) {
             model.addElement(element)
         }
         stepDetailsList.model = model
-        val stepName = transformationPlanLocal?.transformationSteps()?.get(currentStepIdRendered - 1)?.name().orEmpty()
+        // skip step 0 (contains supplemental info)
+        val stepName = transformationPlanLocal?.transformationSteps()?.drop(1)?.get(currentStepIdRendered - 1)?.name().orEmpty()
```

Comment:
Do we never want `step 0`? If so, is it possible to `drop` in `setTransformationPlan` (doing this once and only once) instead of every time we call `render`? 

Also -- I'm guessing stepId is 1-indexed for text rendering purposes, but this array is 0-indexed, which is why we have to subtract 1 here. I'm not sure the best way to indicate this (and this is certainly not a blocker for this PR) -- just something to think about for the future

Reply:
Never want step 0 to display and yes noted, we can think about that more in future.